### PR TITLE
Add leepfrog test

### PR DESCRIPTION
**Pull request by DmitriSimakin**
Добавил херню

### CODE REVIEW

File: test_leepfrog.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+fginwfnf
+gwnigwgnw
+def fegje() 
+   nwiegn
+print("")
```
Comment (dsbarinov1) [inline test_leepfrog.py:5]: Why printing the empty string?

File: test_leepfrog.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+fginwfnf
+gwnigwgnw
+def fegje()
```
Comment (dsbarinov1) [inline test_leepfrog.py]: This function declared in a wrong way. 
```suggestion
def fegje():
  # DO SMTHNG
  return 0
```

File: test_leepfrog.py (hunk 1 above)
Comment (DmitriSimakin) [reply test_leepfrog.py:5]: Because of what

**Review by dsbarinov1 — COMMENTED** [review]
LGTM!

**Review by dsbarinov1 — APPROVED** [review]
LGTM!

**Pull request merged**